src/editor/editor.cpp

```diff
@@ -329,6 +334,27 @@ namespace Editor
                         imgui_tileset_n_cols);
             }
 
+
+            if (ImGui::Button("Save Tileset"))
+            {
+                bool success = TilesetSaveToFile("tileset.json", tileset);
+                if (success)
+                {
+                    imgui_tileset_save_notification_text = imgui_tileset_save_notification_text_success;
+                    
+                }
+                else
+                {
+                    imgui_tileset_save_notification_text = imgui_tileset_save_notification_text_failure;
+                }
+                imgui_save_notification_timer = imgui_save_notification_duration_ms;
```

Comment:
probably need to split `imgui_save_notification_timer` into two vars. one for tilemap and one for tileset. Reason is that the notification will appear twice if it just has one.

Actually.... better idea. 

Instead of having the timers and all that jazz, just write the string to stdout and display it in the terminal. Maybe provide a timestamp too.